whisk/api.go

```diff
@@ -146,13 +146,51 @@ type ApiSwagger struct {
     SwaggerName     string    `json:"swagger,omitempty"`
     BasePath        string    `json:"basePath,omitempty"`
     Info            *ApiSwaggerInfo `json:"info,omitempty"`
-    Paths           map[string]map[string]*ApiSwaggerOperation `json:"paths,omitempty"`  // Paths["/a/path"]["get"] -> a generic object
+    Paths           map[string]*ApiSwaggerPath `json:"paths,omitempty"`  // Paths["/a/path"]["get"] -> a generic object
```

Comment:
i think the comment is incorrect.  should be something like
```// Paths["/a/path"] -> a path (ApiSwaggerPath) object```

Reply:
Removing the comment as I think its no longer required as it was there when the object was a map of maps.